### DISCUSSION TITLE
My cron schedule is not working

### DISCUSSION THREAD

**Discussion by subhrajit-mohanty** [community/community#36204] [Actions]
Even if I gave a schedule for every 5 minutes it's not working.

![image](https://user-images.githubusercontent.com/70169631/195966601-69aaedd4-1b39-4450-a8ea-a76dd84192cf.png)

Is there any other configurations I have to make?

**Comment by ossanna16** [comment]
Hey there @Subhrajit-Katonic and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by antonblr** [comment]
Hi @Subhrajit-Katonic - could you elaborate a bit on "is not working". 

1) Does the job starts at all (i.e. do you see anything at `https://github.com/<OWNER>/<REPO>/actions?query=event%3Aschedule` url)? 
1) If yes, are there any errors?
1) Is that workflow file committed to default branch (`master`, `main` etc)? 

The yaml file looks correct, I have similar setup running on schedule just fine.
